packages/types/src/sdk/search.ts

```diff
@@ -19,6 +19,9 @@ export enum SearchFilterOperator {
 
 export interface SearchFilters {
   allOr?: boolean
+  // TODO: this is just around for now - we need a better way to do or/and
+  // allows just fuzzy to be or - all the fuzzy/like parameters
+  fuzzyOr?: boolean
```

Comment:
Why is this necessary?

Reply:
Trying to create a where statement that allows searching over a variety of fields - discussed this the other day, wish there was a method to specify which fields are or'ed and which fields are and'ed together - but right now you can only signal that you want everything to be OR or you want everything to be AND.

The case for audit logs I was trying to achieve was something like `name LIKE %something% or metadata LIKE %something%` but I still need other props (like some other filter) to be AND on the end. Adding this property allows you to create a really fuzzy search across a bunch of different properties, similar to what we were doing with our Lucene search, essentially achieving feature parity with how the audit log search worked.

Again ideally in the future we would build into our search filters the ability to specific which fields are logically combined together, but its outside the scope of this PR and adding a flag to allow it provided a simple solution until this problem is properly solved.

Arguably Lucene was actually better for this purpose than SQL, but this achieves very similar results (enough that any lack of parity isn't noticeable, its just a free text search box used).